charts/transaction-tool/templates/secrets.yaml

```diff
@@ -41,41 +15,39 @@ metadata:
   labels: {{- include "common.labels" . | nindent 4 }}
 type: kubernetes.io/basic-auth
 data:
-  username: ZGV2ZWxvcG1lbnQ=
-  password: {{ $randomUserPassword }}
----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: {{ .Release.Namespace }}-db-superuser
-  namespace: {{ .Release.Namespace }}
-  annotations:
-    "helm.sh/resource-policy": keep
-{{- if .Values.global.annotations }}
-    {{- toYaml .Values.global.annotations | nindent 4 }}
+  dbusername: dHJhbnNhY3Rpb24tdG9vbA==
+  dbpassword: {{ $b64UserPassword }}
+  dbname: dHJhbnNhY3Rpb24tdG9vbA==
+  script.sql: |
```

Comment:
does this only run once, or every startup?

Just curious if it will cause errors on pod restarts or anything like that.